back/src/forms/validation.ts

```diff
@@ -592,7 +592,10 @@ const recipientSchemaFn: FactorySchemaOf<FormValidationContext, Recipient> = ({
       .string()
       .label("Opération d’élimination / valorisation")
       .ensure()
-      .requiredIf(!isDraft)
+      .when("recipientIsTempStorage", {
+        is: recipientIsTempStorage => !recipientIsTempStorage,
```

Comment:
Peut-être plus élégant comme ça : 

```suggestion
        is: false,
```

Reply:
Mais ça ne couvrirait pas les cas où `recipientIsTempStorage` est null ou undefined :(

Reply:
En effet